src/components/AddUserForm/AddUserForm.js

```diff
@@ -0,0 +1,177 @@
+import React, {Component, createRef} from 'react';
+import uniqId from 'uniqid';
+import {Button} from '../Button/Button';
+import './AddUserForm.scss';
+import imageAttention from '../../assets/icon-attention-png-clip-art.png'
+
+const CN = 'user-form';
+const EMN = {
```

Comment:
The name of the constant should be obvious =) I can't understand what these 3 letters mean. Could you please rename it so it would be clear for what this constant is responsible?